src/Form/LegacyConsumer/ServiceProvider.php

```diff
@@ -27,8 +27,9 @@ function() {
 	 */
 	public function boot() {
 		give( TemplateHooks::class )->walk( give( Commands\SetupNewTemplateHook::class ) );
-		give( TemplateHooks::class )->walk( give( Commands\SetupFieldPersistance::class ) );
 		give( TemplateHooks::class )->walk( give( Commands\SetupFieldValidation::class ) );
+		give( TemplateHooks::class )->walk( give( Commands\SetupFieldPersistance::class ) );
+		give( TemplateHooks::class )->walk( new Commands\CommandFactory( Commands\SetupPaymentDetailsDisplay::class ) );
```

Comment:
I plan to expand use of the `CommandFactory` to other commands to cleanup the nested callbacks.